lib/kennel/models/dashboard.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+module Kennel
+  module Models
+    class Dashboard < Base
+      include TemplateVariables
+      include OptionalValidations
+
+      API_LIST_INCOMPLETE = true
+      DASHBOARD_DEFAULTS = { template_variables: [] }.freeze
+      READONLY_ATTRIBUTES = Base::READONLY_ATTRIBUTES + [
+        :author_handle, :author_name, :modified_at, :url, :is_read_only, :notify_list
+      ]
+      REQUEST_DEFAULTS = {
+        style: { line_width: "normal", palette: "dog_classic", line_type: "solid" }
+      }.freeze
+
+      settings :id, :title, :description, :widgets, :kennel_id, :layout_type
+
+      defaults(
+        description: -> { "" },
+        widgets: -> { [] },
+        id: -> { nil }
```

Comment:
default `layout_type`?

Reply:
you got to pick either dash or screen just like the datadog UI :D